.travis.yml

```diff
@@ -8,15 +8,20 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  - gas -skip=management/examples/*.go -skip=*vendor* ./... | tee gas-scan.txt
+  - cat gas-scan.txt
+  - test -z "$((grep 'Severity:\s*HIGH' gas-scan.txt) -eq 0)"
```

Comment:
also extra parens not required. `-z` is testing empty string. that's why `-eq 0` is not needed anymore as grep output is supposed to be empty.